### PR TITLE
add new location;

### PR DESCRIPTION
**Pull request by Chloejay**
# PyLadies Pull Request
_Please document the following. We will aim to be as timely as possible in processing your pull request, however as this is a volunteer staffed project anticipate at least a week for pull requests to start to be reviewed._ 

## Is this pull requesting adding or updating a [PyLadies chapter website](https://github.com/pyladies/pyladies#to-add-a-new-pyladies-location)? 
 

- [ ] Yes
- [x] No

## Have you confirmed the website builds locally? See [here](https://github.com/pyladies/pyladies#run-the-site-locally).

- [x] Yes
- [ ] No

##  Does your pull request pass the `netlify/pyladies/deploy-preview` check?

- [x] Yes
- [ ] No

# Pull Request Checklist

- [x] Fill out all sections above
- [x] Share pull request in PyLadies Slack channel #project-tech-infra

/cc @pyladies/tech-and-infra-admins

### CODE REVIEW

File: www/config.yml [hunk 1]
```diff
@@ -1369,3 +1369,13 @@ subtitle: Women Who Love Coding in Python
 tag_layout: tag.html
 tags_url: /tags/
 title: PyLadies
+email: singapore@pyladies
```
Comment (Mariatta) [inline www/config.yml:1372]:
```suggestion
email: singapore@pyladies.com
```

File: www/config.yml [hunk 2]
```diff
@@ -1369,3 +1369,13 @@ subtitle: Women Who Love Coding in Python
...
+image: pyladies-sg
+location:
+  latitude: 1.29
+  longitude: 103.85 
+meetup: pyladiesSingapore
+name: Singapore, Singapore
+organizer: Chloe Ji
+pro_network: Python Software Foundation Meetup Pro Network
+twitter: '@pyladies_singapore'
```
Comment (Mariatta) [inline www/config.yml:1381]:
```suggestion
twitter: '@pyladiesSG'

```

**Comment by Chloejay** [comment]
@Mariatta can you help review/merge this?

File: www/config.yml [hunk 3]
```diff
@@ -1369,3 +1369,13 @@ subtitle: Women Who Love Coding in Python
...
+email: singapore@pyladies.com
+image: pyladies-sg
+location:
+  latitude: 1.29
+  longitude: 103.85 
+meetup: pyladiesSingapore
+name: Singapore, Singapore
+organizer: Chloe Ji
+pro_network: Python Software Foundation Meetup Pro Network
+twitter: '@pyladiesSG'
```
Comment (Mariatta) [inline www/config.yml:1381]:
```suggestion
twitter: @pyladiesSG
```

File: www/config.yml [hunk 4]
```diff
@@ -1369,3 +1369,13 @@ subtitle: Women Who Love Coding in Python
 tag_layout: tag.html
 tags_url: /tags/
 title: PyLadies
+email: singapore@pyladies.com
+image: pyladies-sg
```
Comment (Mariatta) [inline www/config.yml:1373]: Did you upload the file? It should be added to /www/_assets/images directory

And this should be a filename i.e pyladies-sg.jpeg or something like that. 
Please see examples from previous entries above.

Comment (Mariatta) [inline www/config.yml:1373]: If there is no image, please remove the line.

Comment (Mariatta) [inline www/config.yml:1373]:
```suggestion
```

File: www/_assets/images/pyladies-sg.jpg
Comment (Mariatta) [inline www/_assets/images/pyladies-sg.jpg:1]: Sorry but please remove the placeholder image. You can upload the actual image later. I don't want to cause confusion and present inaccurate image to the community.

File: www/config.yml [hunk 5]
```diff
@@ -1369,3 +1369,12 @@ subtitle: Women Who Love Coding in Python
...
 title: PyLadies
+email: singapore@pyladies.com
+location:
+  latitude: 1.29
+  longitude: 103.85 
+meetup: pyladiesSingapore
+name: Singapore, Singapore
+organizer: Chloe Ji
+pro_network: Python Software Foundation Meetup Pro Network
+twitter: '@pyladiesSG'
```
Comment (Mariatta) [inline www/config.yml:1380]: Without the `@`. Please check examples from other entries.

**Pull request closed without merging**